### ISSUE TITLE
[BUG] Private member `platform._supported_dists` no longer available in Python 3.8.2

### ISSUE THREAD

**Issue by brianthelion** [saltstack/salt#56752]
The grains implementation relies on a private attribute that's no longer available in Python 3.8.2. See #55310 and #55835

```bash
salt --versions-report
Traceback (most recent call last):
  File "/home/br/Dev/f0cal/2020-04/_venv/bin/salt", line 8, in <module>
    sys.exit(salt_main())
  File "/home/br/Dev/f0cal/2020-04/_venv/lib/python3.8/site-packages/salt/scripts.py", line 525, in salt_main
    import salt.cli.salt
  File "/home/br/Dev/f0cal/2020-04/_venv/lib/python3.8/site-packages/salt/cli/salt.py", line 11, in <module>
    import salt.utils.job
  File "/home/br/Dev/f0cal/2020-04/_venv/lib/python3.8/site-packages/salt/utils/job.py", line 11, in <module>
    import salt.minion
  File "/home/br/Dev/f0cal/2020-04/_venv/lib/python3.8/site-packages/salt/minion.py", line 34, in <module>
    import salt.utils.crypt
  File "/home/br/Dev/f0cal/2020-04/_venv/lib/python3.8/site-packages/salt/utils/crypt.py", line 16, in <module>
    import salt.loader
  File "/home/br/Dev/f0cal/2020-04/_venv/lib/python3.8/site-packages/salt/loader.py", line 24, in <module>
    import salt.config
  File "/home/br/Dev/f0cal/2020-04/_venv/lib/python3.8/site-packages/salt/config/__init__.py", line 100, in <module>
    _DFLT_IPC_WBUFFER = _gather_buffer_space() * .5
  File "/home/br/Dev/f0cal/2020-04/_venv/lib/python3.8/site-packages/salt/config/__init__.py", line 89, in _gather_buffer_space
    import salt.grains.core
  File "/home/br/Dev/f0cal/2020-04/_venv/lib/python3.8/site-packages/salt/grains/core.py", line 40, in <module>
    from platform import _supported_dists
ImportError: cannot import name '_supported_dists' from 'platform' (/usr/lib/python3.8/platform.py)
```

Despite https://github.com/saltstack/salt/pull/56031#issuecomment-614324858, this has not been fixed on `master` of today:

https://github.com/saltstack/salt/blob/5ad847d619c5c80873d0b67f2ffdeff00323968c/salt/grains/core.py#L28-L32

**Comment by alexey-zhukovin** [comment]
Thank you! 
It will be fixed after #55310

duplicate of #55310 closing
